ImageManager/ImageManager.py

```diff
@@ -1,35 +1,50 @@
 """
     Handles CRUD operations for firmware binary images
+    Each image consists of an metadata entry and optionally an binary file
+    Binary files are stored temporarily at /tmp/. This implementation assumes /tmp/ is cleaned on a regular basis
+    and no effort to remove temporary files are made.
 """
 
 import json
 import logging
-from time import time
+import os
+import uuid
 from flask import request
-from flask import make_response
-from flask import redirect, url_for
+from flask import send_from_directory
 from flask import Blueprint
-from .utils import *
+from minio.error import ResponseError
 
+from .utils import *
 from .DatabaseModels import *
 from .SerializationModels import *
 from .TenancyManager import init_tenant_context
-
 from .app import app
 
-import uuid
-
 image = Blueprint('image', __name__)
 
 LOGGER = logging.getLogger('image-manager.' + __name__)
 LOGGER.addHandler(logging.StreamHandler())
 LOGGER.setLevel(logging.DEBUG)
 
 
+@image.route('/image/', methods=['GET'])
+def get_all():
+    try:
+        init_tenant_context(request, db, minioClient)
+        images = get_all_images()
+        json_images = [image_schema.dump(i).data for i in images]
```

Comment:
I had a few problems with dump(x).data with the new Marshmallow version - apparently, dump() returns the dict directly (without data attribute)

Reply:
Yeah, I missed that.
Fixed